apps/server/src/shared/domain/entity/user.entity.ts

```diff
@@ -27,7 +27,7 @@ export interface IUserProperties {
 @Entity({ tableName: 'users' })
 @Index({ properties: ['id', 'email'] })
 @Index({ properties: ['firstName', 'lastName'] })
-@Index({ properties: ['ldapId', 'school'] })
+@Index({ properties: ['externalId', 'school'] })
```

Comment:
Do you need the columnName or the propertyName for the index? check this please

Reply:
we need the property name. At least that is what the error message says, if you use the column name.